### ISSUE TITLE
Cluster no supports browserWSEndpoint

### ISSUE THREAD

**Issue by gvojtko** [thomasdondorf/puppeteer-cluster#324]
I can not connect the browser to external ws endpoit. PuppeteerOptions still no supports browserWSEndpoint. It is possible to use Cluster.connect like puppeteer.connect?

**Comment by gvojtko** [comment]
Still doesn't support wsEndpoint?

**Comment by nkostadinov** [comment]
why do you need a cluster with wsendpoint ?

**Comment by gvojtko** [comment]
I have created own browserless nodes to which I want to send requests.

**Comment by nkostadinov** [comment]
Yes, but the clusterization should be done in the nodes. And in the client you just call it

**Comment by gvojtko** [comment]
I understand. Nevertheless, for various reasons, we are more comfortable using Browserless nodes. That's why I created my own queue for consuming urls.

**Comment by pilot114** [comment]
is it possible now to use puppeteer-cluster with browserless (https://www.browserless.io)?

**Comment by vukhaihoan** [comment]
I made this package by fork puppeteer-cluster : https://github.com/vukhaihoan/puppeteer-cluster-connect
you can install by command :  `npm i puppeteer-cluster-connect`
Current support one browser options ( I haven't tested when connecting with multiple browsers at the same time so I'm not sure if it will work )

**Comment by EarthlingDavey** [comment]
Hi @vukhaihoan thank you for your fork.

It looks legit, and I'm using it in development. Incase anyone wants to compare the changes here's a quick view.

https://github.com/thomasdondorf/puppeteer-cluster/compare/b5b098a...vukhaihoan:master

**Comment by EarthlingDavey** [comment]
Hi @thomasdondorf . First off thanks for the awesome project. It's realy useful, as my previous alternative was a mess of loops and retry counters and browser restarts.

I've been using it with browserless (self hosted) recently. Would you be interested in merging @vukhaihoan 's [work](https://github.com/thomasdondorf/puppeteer-cluster/compare/b5b098a...vukhaihoan:master) (with some additions from me)?
